JamieOliver: typo in image retrieval method

**Pre-filing checks**

- [x] I have searched for open issues that report the same problem
- [x] I have checked that the bug affects the latest version of the library

**The URL of the recipe(s) that are not being scraped correctly**

- https://www.jamieoliver.com/recipes/chocolate-recipes/bloomin-brilliant-brownies/

**The results you expect to see**

The `image` value expectation in the [test data](https://github.com/hhursev/recipe-scrapers/blob/117081c43e2d8703815af60a1e4413a36af00110/tests/test_data/jamieoliver.com/jamieoliver.json) seems incorrect here; the code looks intended to retrieve data from `schema.org`, but because of the typo in the method name, it raises an exception and uses a plugin fallback during the unit tests.

**The results (including any Python error messages) that you are seeing**

After fixing the typo:

```
AssertionError: 'https://cdn.jamieoliver.com/recipe-database/oldIm[27 chars].jpg' != 'https://img.jamieoliver.com/jamieoliver/recipe-da[53 chars]-800'
- https://cdn.jamieoliver.com/recipe-database/oldImages/medium/88_1_1441269331.jpg
?         ^^^                                           ^ ----
+ https://img.jamieoliver.com/jamieoliver/recipe-database/oldImages/large/88_1_1441269331.jpg?tr=w-800,h-800
?         ^^^                ++++++++++++                           ^^^^                     +++++++++++++++
 : The actual value for .image() did not match the expected value.
```

--- a/recipe_scrapers/jamieoliver.py
+++ b/recipe_scrapers/jamieoliver.py
@@ -9,7 +9,7 @@ class JamieOliver(AbstractScraper):
         return "jamieoliver.com"
 
     def image(self):
-        return self.schema.catimageegory()
+        return self.schema.image()
 
     def instructions(self):
         instructions = self.soup.find("ol", {"class": "recipeSteps"}).findAll("li")

--- a/tests/test_data/jamieoliver.com/jamieoliver.json
+++ b/tests/test_data/jamieoliver.com/jamieoliver.json
@@ -3,7 +3,7 @@
   "canonical_url": "https://www.jamieoliver.com/recipes/chocolate-recipes/bloomin-brilliant-brownies/",
   "category": "Desserts",
   "host": "jamieoliver.com",
-  "image": "https://cdn.jamieoliver.com/recipe-database/oldImages/medium/88_1_1441269331.jpg",
+  "image": "https://img.jamieoliver.com/jamieoliver/recipe-database/oldImages/large/88_1_1441269331.jpg?tr=w-800,h-800",
   "ingredient_groups": [
     {
       "ingredients": [

